libosmscout/src/osmscout/ost/Parser.cpp

```diff
@@ -412,7 +412,7 @@ void Parser::TYPEKINDS(unsigned char& types) {
 }
 
 void Parser::TAGCONDITION(TagConditionRef& condition) {
-		std::list<TagConditionRef> conditions;
+		std::vector<TagConditionRef> conditions;
```

Comment:
This change is fine in general. And as import definition is parsed just once, usage of vector is better IMHO (but would be great to confirm it by some bechmark). But problem is that this file is generated by this script: https://github.com/Framstag/libosmscout/blob/master/libosmscout-map/parser/generate.sh (manually, after definition change)